### PR TITLE
RF POM /TC_02.02.003 | Verify visibility of Jenkins logo and title on Dashboard page

### PR DESCRIPTION
**Pull request by Nataly-Mayatskaya**
https://trello.com/c/CxKKQyH3/410-rf-pom-tc0202003-verify-visibility-of-jenkins-logo-and-title-on-dashboard-page

### CODE REVIEW

**Review by Bainoff — APPROVED** [review]

File: cypress/pageObjects/HomePage.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 class HomePage {
     getNewItemLink = () => cy.get('a[href="newJob"]');
     getDashboardBreadcrumbsLink = () => cy.get('li.jenkins-breadcrumbs__list-item a[href="/"]');
     getProjectNameLink = () => cy.get('td a[href*="job"].jenkins-table__link');
     getBuildHistoryLink = () => cy.get('td:last-child [tooltip]'); 
     getRestApilink = () => cy.get('.rest-api');
-
-
+    getJenkinsLogo = () => cy.get('#jenkins-head-icon');
+    getJenkinsTitle = () =>  cy.get('#jenkins-name-icon');
```
Comment (Rom-T) [inline cypress/pageObjects/HomePage.js:14]: These locators should not be on the HomePage, but on the HeaderPage.

File: cypress/e2e/pom_e2e/verifyLogoAndTitle.cy.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        homePage
+            .clickPeopleLink()
+
+        peoplePage
+            .clickJenkinsHomeLink()
+
+        homePage.getWellcomeTitle()
+            .should('be.visible')
+            .and('have.text', homePageData.greeting)
```
Comment (Rom-T) [inline cypress/e2e/pom_e2e/verifyLogoAndTitle.cy.js:27]: It’s better to do it in the form of a chain.
`homePage.clickPeopleLink()
        .clickJenkinsHomeLink()
        .getWellcomeTitle()
        .should('be.visible')
        .and('have.text', homePageData.greeting)`

File: cypress/e2e/pom_e2e/verifyLogoAndTitle.cy.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (Rom-T) [inline cypress/e2e/pom_e2e/verifyLogoAndTitle.cy.js:28]: Although I don’t really understand why you are checking this title in this TC. After all, the main task is to check the logo and its title.

Comment (Nataly-Mayatskaya) [reply cypress/e2e/pom_e2e/verifyLogoAndTitle.cy.js:24]: fixed

**Review by Paul-Korz — APPROVED** [review]

**Pull request merged**